.drone.yml

```diff
@@ -74,15 +73,62 @@ pipeline:
       matrix:
         TEST_SUITE: owncloud-coding-standard
 
+  phpstan:
+    # phpstan requires php7.1
+    image: owncloudci/php:7.1
+    pull: true
+    environment:
+      - COMPOSER_HOME=/var/www/owncloud/apps/password_policy/.cache/composer
+    commands:
+      - make test-php-phpstan
+    when:
+      matrix:
+        TEST_SUITE: phpstan
+
+  phan-70:
+    # phan requires a recent php-ast extension
+    image: owncloudci/php:7.0
```

Comment:
why not using ${PHP_VERSION} here?

Reply:
There is a ``phan`` pipeline step specifically for each PHP version. That allows a single matrix entry to run ``phan`` for a list of different PHP versions. @patrickjahns suggested that.

Now there is a special case for PHP 7.0 which has to be run with core ``stable10`` - so there needs to be a separate matrix entry for that.